src/components/common/Stepper.vue

```diff
@@ -79,18 +78,18 @@ export default class Stepper extends Vue {
       case RouteNames.INCORPORATION_DEFINE_COMPANY: return this.isDefineCompanyValid
       case RouteNames.INCORPORATION_MEMORANDUM: return this.isMemorandumValid
       case RouteNames.INCORPORATION_PEOPLE_ROLES: return this.isAddPeopleAndRolesValid
-      case RouteNames.INCORPORATION_REVIEW_CONFIRM: return this.isApplicationValid
+      case RouteNames.INCORPORATION_REVIEW_CONFIRM: return this.isIncorporationApplicationValid
       case RouteNames.INCORPORATION_RULES: return this.isRulesValid
       case RouteNames.INCORPORATION_SHARE_STRUCTURE: return this.isCreateShareStructureValid
 
       case RouteNames.DISSOLUTION_AFFIDAVIT: return this.isAffidavitValid
       case RouteNames.DISSOLUTION_DEFINE_DISSOLUTION: return this.isDissolutionDefineDissolutionValid
       case RouteNames.DISSOLUTION_RESOLUTION: return this.isResolutionValid
-      case RouteNames.DISSOLUTION_REVIEW_CONFIRM: return this.isApplicationValid
+      case RouteNames.DISSOLUTION_REVIEW_CONFIRM: return this.isDissolutionValid
 
       case RouteNames.REGISTRATION_DEFINE_BUSINESS: return this.getRegistration.defineBusinessValid
       case RouteNames.REGISTRATION_PEOPLE_ROLES: return this.isAddPeopleAndRolesValid
-      case RouteNames.REGISTRATION_REVIEW_CONFIRM: return this.isApplicationValid
+      case RouteNames.REGISTRATION_REVIEW_CONFIRM: return this.isRegistrationValid
```

Comment:
This worked before because `isRegistrationValid` would return the correct value for the current filing type, but that's a bit obscure. This is more understandable.